aws/resource_aws_sfn_state_machine_test.go

```diff
@@ -299,19 +304,22 @@ resource "aws_sfn_state_machine" "test" {
       "Resource": "${aws_lambda_function.test.arn}",
       "Retry": [
         {
-          "ErrorEquals": ["States.ALL"],
+          "ErrorEquals": [
+            "States.ALL"
+          ],
```

Comment:
This could be left as one line

Reply:
"Properly" formatted JSON has this split out....